Production/govuk_ios/Model/RecentActivity/ActivityItem.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import CoreData
+
+@objc(ActivityItem)
+class ActivityItem: NSManagedObject, Identifiable, GroupedListRow {
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<ActivityItem> {
+        let request: NSFetchRequest<ActivityItem> = .init(entityName: "ActivityItem")
+        request.sortDescriptors = [
+            NSSortDescriptor(
+                keyPath: \ActivityItem.date,
+                ascending: false)
+        ]
+        return request
+    }
+
+    @NSManaged var id: String
+    @NSManaged var title: String
+    @NSManaged var date: Date
+    @NSManaged var url: String
+
+    var formattedDate: String {
+        let day = DateHelper.getDay(date: date)
+        guard let day = day else { return ""}
+        let components = DateHelper.returnCalanderComponent(date: date)
+        let month = DateHelper.getMonthName(components: components)
+        let dateString = ("\(formattedString) \(day) \(month)")
```

Comment:
This seems a little convoluted, is there a reason not to use DateFormatter here?
Also, I'm not sure this should live in the `ActivityItem` it's quite opinionated about where the date is being displayed.